queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
* Kindly use the right query for this question instead of '%mon%' which will look for 'mon' anywhere in the string( animal name). you should use this instead  '%mon' this fetches animal name that end with the string 'mon' 👍

Reply:
Thanks for your suggestion. :+1: